packages/react-scripts/template/src/service-worker-registration.js

```diff
@@ -0,0 +1,38 @@
+export default function register() {
```

Comment:
Technically `react-dev-utils` contains a mix of things that are useful either in Node or web context. Since every entry point is separate, it’s fine to add things that are browser-only there.

On the other hand, I don’t think we really want the generated app to depend on `react-dev-utils` explicitly, and doing so implicitly (relying on npm 3 flattening) is more confusing. So I’d leave it as is.